src/views/setup/Install.vue

```diff
@@ -68,24 +130,19 @@ function setToolsFolder(newToolsPath: string) {
         :openMethod="store.openEspIdfToolsFolder"
       />
 
-      <selectPyVersion v-if="isNotWinPlatform"></selectPyVersion>
-
-      <div
-        class="notification is-danger error-message"
-        v-if="pyExecErrorStatus"
-      >
-        <p>{{ pyExecErrorStatus }}</p>
```

Comment:
Why are you deleting the `pyExecErrorStatus` section ? I don't see how are these errors from SetupPanel.ts are managed

```ts
else if (
      errMsg.indexOf("ERROR_INVALID_PYTHON") !== -1 ||
      errMsg.indexOf("ERROR_INVALID_PIP") !== -1 ||
      errMsg.indexOf("ERROR_INVALID_VENV") !== -1 ||
      errMsg.indexOf("PYTHON_BIN_PATH_WITH_SPACES") !== -1
    ) {
      SetupPanel.postMessage({
        command: "setPyExecErrorStatus",
        errorMsg: error.message,
      });
```

Reply:
Hi @brianignacio5, sorry for the messy commits.

I didn't delete them, I just moved them above. I've considered since these errors are created when user is returned to the install web view during installation, that having them as first element on the page will make the most sense.

Reply:
Ah my bad makes sense.